src/daemon/scheduler.rs

```diff
@@ -208,6 +208,7 @@ impl Scheduler {
         }
 
         self.tasks.republish_if_needed(now());
+        self.tasks.renew_if_needed(now());
```

Comment:
Note that when this task is done, it will re-schedule itself - just like the republish_if_needed does.